Buttons/SelfPacedExercise.md

```diff
@@ -0,0 +1,55 @@
+
+All individuals exposed to `SCP-3997` are to be placed under immediate quarantine and monitored for symptoms of `SCP-3997-A`. If symptoms are detected, the affected individual is to be treated with Class-B amnestics and removed from `SCP-3997`-related projects.
+
+**Description:** `SCP-3997` is a website, created in the late 1990s, dedicated to the "Tickle Me Elmo" doll. While the website's interface appears non-anomalous, the source code of the site's Tickle button contains a complex and obfuscated Javascript function that, when analyzed, produces a series of increasingly disturbing effects on the viewer, collectively known as `SCP-3997-A`.
+
+## `SCP-3997-A`
+
+Symptoms of `SCP-3997-A` manifest in stages, as outlined below:
+
+**Stage 1:** Initial exposure to `SCP-3997`'s source code results in mild unease and discomfort, often accompanied by a feeling of being watched. Subjects at this stage have reported a sense of obligation to fix the Javascript code in order to restore the Tickle button functionality.
+
+**Stage 2:** After approximately one hour of continuous exposure, subjects begin to exhibit signs of increased agitation and an obsessive fixation on the Tickle Me Elmo toy. They may show heightened sensitivity to any mention or image of Elmo, becoming defensive or aggressive when questioned about their fixation.
+
+**Stage 3:** At this stage, subjects begin to experience auditory hallucinations, often described as faint giggling or ██████████. These hallucinations increase in intensity over time, causing subjects to become more determined to "unlock" the laughter contained within `SCP-3997`'s code.
+
+**Stage 4:** Subjects will develop a strong aversion to any form of laughter or mirth, perceiving it as mocking or sinister. This aversion may escalate to the point where subjects become combative or hostile when exposed to laughter or light-hearted situations.
+
+**Stage 5:** In the final stage, subjects become entirely unresponsive to external stimuli, fixating solely on the screen displaying `SCP-3997`. They will refuse to eat, drink, or sleep, remaining transfixed on the website until their eventual expiration due to dehydration or exhaustion.
+
+## Addendum `3997-1`: Initial Analysis of `SCP-3997`
+
+During the initial analysis of `SCP-3997`, research personnel Dr. A█████ and Dr. S███████ attempted to correct the dysfunctional source code, believing that fixing the Tickle button would mitigate the effects of `SCP-3997-A`. Several anomalies within the code were identified and documented:
+
+<input type="checkbox" id="reveal1" class="reveal-checkbox" />
+
+<label for="reveal1" class="reveal-label">👀 Click to Reveal Anomalies 👀</label>
+
+```text
+> SCP-3997/588364/AB: The script tag in the HTML file contained a misspelled attribute: scr instead of src.
```

Comment:
Love this idea overall. I am wondering if there are too many bugs.. maybe we could make multiple elmos, each one with one bug? that might be a lot, but the students would get a quicker reward which might be more encouraging. not sure.

Reply:
There are a ton of bugs, yeah, and one that I bet nobody will get without looking at the help document (the Turkish ı). I do like the idea of splitting this into multiple exercises, but I do not want to do that work 😄. GPT, however... 🤔

You're right about this style not being particularly rewarding though. I hope that since we're giving them the answers they'll be able to get through it anyway 🤔